dashboard/lib/services/lti.rb

```diff
@@ -299,5 +299,12 @@ def self.sync_course_roster(lti_integration:, lti_course:, nrps_sections:, curre
       end
       course_sync_result
     end
+
+    def self.initialize_lms_landing_session(session, lti_provider_name, new_cta_type)
+      session[:lms_landing] = {
+        lti_provider_name: lti_provider_name,
```

Comment:
minor nit: could change this to `lti_provider` to match the name of the param in the view